lib/jxl/modular/encoding/encoding.cc

```diff
@@ -479,6 +478,7 @@ Status ModularDecode(BitReader *br, Image &image, GroupHeader &header,
     if (!br->AllReadsWithinBounds()) {
       if (!allow_truncated_group) return JXL_FAILURE("Truncated input");
       ZeroFillImage(&channel.plane);
+      while (++i < nb_channels) ZeroFillImage(&image.channel[i].plane);
```

Comment:
This is a bugfix I think — there could be cases where channels were not initialized nor decoded, yet their values would be used when showing a progressive preview in the `allow_truncated_group == true` case.